packages/server-wallet/src/models/funding.ts

```diff
@@ -0,0 +1,59 @@
+import {Model, Transaction} from 'objection';
+import {BigNumberish} from 'ethers';
+import {Zero} from '@statechannels/wallet-core';
+
+import {Uint256, Bytes32, Address} from '../type-aliases';
+
+export const REQUIRED_COLUMNS = {
+  channelId: 'channelId',
+  amount: 'amount',
+  assetHolder: 'assetHolder',
+};
+export interface RequiredColumns {
+  readonly channelId: Bytes32;
+  readonly amount: Uint256;
+  readonly assetHolder: Address;
+}
+
+export class Funding extends Model implements RequiredColumns {
+  static tableName = 'funding';
+  static get idColumn(): string[] {
+    return ['channelId', 'assetHolder'];
+  }
+  readonly channelId!: Bytes32;
+  readonly amount!: Uint256;
+  readonly assetHolder!: Address;
+
+  static async getFundingAmount(
+    channelId: Bytes32,
+    assetHolder: Address,
+    tx: Transaction | undefined
+  ): Promise<BigNumberish> {
```

Comment:
I don't think we should use `BigNumberish` -- we should return a `Uint256`, which anyone can consume as a `string`.

The only runtime type returned by `getFundingAmount` is a `string`.